src/index.ts

```diff
@@ -1 +1,7 @@
-export { default as BeaconDetector } from './BeaconDetector';
+import os from 'os';
+import BeaconDetectorMac from './BeaconDetector.mac';
```

Comment:
won't this fail at import time when the wrong platform is imported? I think TS supports dynamic imports, maybe we can solve it that way?

Reply:
You know, for me it's just 12 or a dozen :)